code/controllers/configuration/entries/game_options.dm

```diff
@@ -214,6 +214,7 @@
 /datum/config_entry/keyed_list/multiplicative_movespeed
 	key_mode = KEY_MODE_TYPE
 	value_mode = VALUE_MODE_NUM
+	integer = TRUE
```

Comment:
are you sure setting integer to TRUE does not in fact make it an integer

Reply:
WAIT NO FUCK